### PR TITLE
os/bluestore: no need to fsync when failed to write label

### PR DESCRIPTION
**Pull request by tangwenjun3**
Signed-off-by: tangwenjun <tang.wenjun3@zte.com.cn>

### CODE REVIEW

File: src/os/bluestore/BlueStore.cc [hunk 1]
```diff
@@ -4160,6 +4160,8 @@ int BlueStore::_write_bdev_label(CephContext *cct,
   if (r < 0) {
     derr << __func__ << " failed to write to " << path
 	 << ": " << cpp_strerror(r) << dendl;
+    VOID_TEMP_FAILURE_RETRY(::close(fd));
```
Comment (liewegas) [inline src/os/bluestore/BlueStore.cc]: Instead, can you 'goto out', and add an out: label on the VOID_TEMP_FAILURE... line below?

**Comment by tangwenjun3** [comment]
@liewegas fixed

**Comment by liewegas** [comment]
thanks, looks good!

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2018-01-28_09:53:35-rados-wip-kefu-testing-2018-01-27-2356-distro-basic-mira/

the failed tests are tracked by

- http://tracker.ceph.com/issues/20086
- http://tracker.ceph.com/issues/21312

**Pull request merged**